proteus/SpatialTools.py

```diff
@@ -0,0 +1,1469 @@
+"""
```

Comment:
Yes I agree, that would be better. We could also keep classes such as Rectangle, Cuboid, CustomShape in here and some general functions that could be useful (e.g. rotate/translate), but make a module in mprans that would import everything from here, and add functionality to it related to two-phase flow and floating bodies to the Shapes (e.g. setAbsorptionZones, setRigidBody, etc). Let me know what you think.